commit/merkleroot/outcome.go

```diff
@@ -227,11 +235,15 @@ func buildMerkleRootsOutcome(
 				OnRampAddress: typconv.AddressBytesToString(root.OnRampAddress, uint64(root.ChainSel)),
 			}
 
-			if signedRoots.Contains(rk) {
+			switch {
+			case signedRoots.Contains(rk):
 				lggr.Infow("Root is signed, appending to the report", "root", rk)
 				rootsToReport = append(rootsToReport, root)
-			} else {
-				lggr.Infow("Root not signed by RMN, skipping from the report", "root", rk)
+			case !consensusObservation.RMNEnabledChains[root.ChainSel]:
+				lggr.Infow("Root is not signed, but RMN is disabled for the chain, appending to the report", "root", rk)
+				rootsToReport = append(rootsToReport, root)
+			default:
+				lggr.Infow("Root not signed, skipping from the report", "root", rk)
 			}
 		}
 		roots = rootsToReport
```

Comment:
Ditto here on aliasing, we can't just append to `roots` directly?

Reply:
Oh I just realized we loop through `roots` above ... is it just me or is this entire if block rly confusing and hard to understand?